src/main/java/frc/robot/Robot.java

```diff
@@ -15,6 +15,7 @@
 import frc.robot.subsystems.shooter.ShooterSpeedController;
 import frc.robot.subsystems.drive.Gyroscope;
 import frc.robot.commands.AutonomousCommand;
+import frc.robot.commands.CurvatureDriveCommand;
 import frc.robot.commands.LightsCommand;
 import frc.robot.commands.TeleopCommand;
```

Comment:
Rename TeleopCommand to ArcadeDriveCommand.

Reply:
done